src/plugins/workspace/public/components/workspace_updater/workspace_updater.tsx

```diff
@@ -129,7 +127,54 @@ export const WorkspaceUpdater = () => {
       }
     }
     setDeleteWorkspaceModalVisible(false);
-    await application.navigateToApp('home');
+    window.location.href =
+      workspaces?.formatUrlWithWorkspaceId(
```

Comment:
I can't get the point here. Why do we need to generate a workspace link with empty workspace id? How about just navigate to 
```js
application.getUrlForApp('home', {
          path: '/',
          absolute: true,
        })
```

Reply:
Exiting workspace through `application.navigateToUrl` would result in wrong target url like:
`http://localhost:5601/w/5786ccb0-26a7-11ee-8cfa-113158c5ccb1/app/home#/`
This is because the workspace id will be part of the url when inside workspace. Do you have any suggestions?

Reply:
Consider use `src/core/public/http/base_path.ts`, combine `remove` and `prepand` without workspace?